bin/zmonlyofficeconfig

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+# 
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2016, 2021 Synacor, Inc.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+# 
+
+source /opt/zimbra/bin/zmshutil || exit 1
+zmsetvars
+
+zmjq="/opt/zimbra/onlyoffice/bin/jq"
+
+defaultjson_path="/opt/zimbra/onlyoffice/documentserver/server/Common/config/default.json"
+mysql_port=7306
+zimbra_mysql_user="zimbra"
+zimbra_mysql_password=""
+server_crt="/opt/zimbra/ssl/zimbra/server/server.crt"
+server_key="/opt/zimbra/ssl/zimbra/server/server.key"
+
+STATUS=0
+
+document_editing_JWT_secret=""
+
+usage() {
+  echo ""
+  echo "Invalid argument. Allowed :"
+  echo "new                     Configure Onlyoffice on the server"
+  echo "update-jwt-secret arg0  Update Onlyoffice server's jwt secret with the passed value"
+  echo "update-db               Update Onlyoffice server's DB connection details from config"
+  echo ""
+  exit
+}
+
+generate_random()
+{
+  length=8
+  digits=({2..9})
+  lower=({a..k} {m..n} {p..z})
+  upper=({A..N} {P..Z})
+  CharArray=(${digits[*]} ${lower[*]} ${upper[*]})
+  ArrayLength=${#CharArray[*]}
+  password=""
+  for i in `seq 1 $length`
```

Comment:
I dont think you are using the the var i here..You could use different way of iterating..

Reply:
It is similar to initialising i to 1 and then check if it is less than 8. Shouldn't make a difference.

Reply:
Check this https://github.com/ZimbraOS/zm-onlyoffice/pull/2#issuecomment-878998536

Reply:
done